src/core/meshcop/meshcop_tlvs.hpp

```diff
@@ -518,18 +519,21 @@ class NetworkMasterKeyTlv : public Tlv
      * @returns The Network Master Key value.
      *
      */
-    const otMasterKey &GetNetworkMasterKey(void) const { return mNetworkMasterKey; }
+    const MasterKey &GetNetworkMasterKey(void) const { return mNetworkMasterKey; }
 
     /**
      * This method sets the Network Master Key value.
      *
-     * @param[in]  aNetworkMasterKey  A pointer to the Network Master Key value.
+     * @param[in]  aMasterKey  The Network Master Key.
      *
      */
-    void SetNetworkMasterKey(const otMasterKey &aNetworkMasterKey) { mNetworkMasterKey = aNetworkMasterKey; }
+    void SetNetworkMasterKey(const otMasterKey &aMasterKey)
```

Comment:
nit: should we use `MasterKey` here instead of `otMasterKey` to be consistent with `GetNetworkMasterKey()`? I realize this will likely require adding `static_cast` at some call sites.